api/desecapi/pdns.py

```diff
@@ -108,3 +133,18 @@ def set_dyn_records(name, a, aaaa):
             _delete_or_replace_rrset(name, 'aaaa', aaaa),
         ]
     })
+
+
+def set_rrset(zone, name, type, value):
```

Comment:
This function is very similar to `set_dyn_records()`. I'd prefer that `set_dyn_records()` uses the `set_rrset()` function to do what needs to be done, however in the current state this is only possible with two pdns api requests, which is certainly not desirable. Should `set_rrset` be extended to `set_rrsets`?